readme.txt

```diff
@@ -69,7 +69,7 @@ Other plugins of Jotform allow you to embed your forms to websites. However, wit
 
 == Changelog ==
 
-= 2.0.0 =
+= 2.0.1 =
```

Comment:
@aycabilikk please do not change this section. Instead add a new one for 2.0.1 with a summary of the change